Themes/default/languages/index.english.php

```diff
@@ -847,7 +847,7 @@
 $txt['debug_queries_used_and_warnings'] = 'Queries used: %1$d, %2$d warnings.';
 $txt['debug_query_in_line'] = 'in <em>%1$s</em> line <em>%2$s</em>, ';
 $txt['debug_query_which_took'] = 'which took %1$s seconds.';
-$txt['debug_query_which_took_at'] = 'which took %1$s seconds at %2$s into request.';
+$txt['debug_query_which_took_at'] = 'which took %1$s seconds in php %2$s in db at %3$s into request.';
```

Comment:
'which took %1$s seconds to prepare in PHP and %2$s to execute in the database at %3$s into request.'